test/functional/Java8andUp/src/org/openj9/test/vmArguments/VmArgumentTests.java

```diff
@@ -719,14 +719,11 @@ public void testEnvironmentVariableOrdering() {
 				argumentPositions.containsKey(openJ9JavaOptionsArg));
 		assertTrue(IBM_JAVA_OPTIONS+ SHOULD_COME_AFTER+OPENJ9_JAVA_OPTIONS, 
 				argumentPositions.get(ibmJavaOptionsArg).intValue() > argumentPositions.get(openJ9JavaOptionsArg).intValue());
-		if (!isJava8 || !isIBM) {
-			assertTrue(OPENJ9_JAVA_OPTIONS + SHOULD_COME_AFTER + JAVA_OPTIONS, 
-					argumentPositions.get(openJ9JavaOptionsArg).intValue() > argumentPositions.get(javaOptionsArg).intValue());
-			assertTrue(JAVA_OPTIONS + SHOULD_COME_AFTER + JAVA_TOOL_OPTIONS, 
-					argumentPositions.get(javaOptionsArg).intValue() > argumentPositions.get(javaToolOptionsArg).intValue());
-		} else {
-			assertTrue(OPENJ9_JAVA_OPTIONS + SHOULD_COME_AFTER + JAVA_TOOL_OPTIONS, 
-					argumentPositions.get(openJ9JavaOptionsArg).intValue() > argumentPositions.get(javaToolOptionsArg).intValue());
+		assertTrue(OPENJ9_JAVA_OPTIONS + SHOULD_COME_AFTER + JAVA_TOOL_OPTIONS, 
+				argumentPositions.get(openJ9JavaOptionsArg).intValue() > argumentPositions.get(javaToolOptionsArg).intValue());
```

Comment:
Is this change intentional? It only matches an old test for `isJava8 && isIBM`.

Reply:
Since JAVA_OPTIONS is removed and _JAVA_OPTIONS is parsed after command line options, OpenJ9 and IBM Java 8 now have the same behaviour, OPENJ9_JAVA_OPTIONS comes after JAVA_TOOL_OPTIONS.

Reply:
OPENJ9_JAVA_OPTIONS has always been after JAVA_TOOL_OPTIONS, but previously it was also after JAVA_OPTIONS (on OpenJ9 builds) which is why the test was conditional.

Reply:
A build that identified as `!isIBM` should not be paying any attention to `OPENJ9_JAVA_OPTIONS`.